src/app/views/panels/widgets/controls/fluentui_image.tsx

```diff
@@ -23,6 +23,7 @@ import {
   FluentActionButton,
   FluentButton,
 } from "./fluentui_customized_components";
+import { ImageMappingDragStateWrapper } from "./styles";
```

Comment:
`import` statement for `ImageMappingDragStateWrapper` is not used in this control. Consider removing the statement.

Reply:
Removed